docs/pages/api-docs/accordion.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
-import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
-import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+import ApiDocs from 'docs/src/modules/components/ApiDocs';
+import mapApiTranslations, { parsePropsMarkdown } from 'docs/src/modules/utils/mapApiTranslations';
+import jsonPageContent from './accordion.json';
 
-const pageFilename = 'api/accordion';
-const requireRaw = require.context('!raw-loader!./', false, /\/accordion\.md$/);
-
-export default function Page({ docs }) {
-  return <MarkdownDocs docs={docs} />;
+export default function Page({ pageContent }) {
+  return <ApiDocs pageContent={pageContent} />;
```

Comment:
What do you think about the `TopLayout` naming convention we have started to use for the blog `TopLayoutBlog` and the marketing pages `TopLayoutCompany`?

```suggestion
  return <TopLayoutApiDocs pageContent={pageContent} />;
```

Reply:
Sure. Incidentally, the layout is identical to `MarkdownDocs.js`. Should I extract that as a wrapper component for both, or duplicate it as it is now?

Reply:
It seems that making an abstraction to share the layout between API and demos pages makes a lot of sense +1/